### DISCUSSION TITLE
SSG doesn't render breadcrumbs reliably

### DISCUSSION THREAD

**Discussion by markzegarelli** [statamic/ssg#169] [Q&A]
Hi all,

I'm using Vercel to build and deploy my site, but breadcrumbs for individual pages seem to be hit or miss. For example, when I work locally, I have a page at: `SDKs / SDK Catalog / Browser / Browser SDK 2`.

The SSG output on Vercel for the same page shows ` SDK Catalog / Browser / `, and drops the first and last pages.

I've tried manually specifying the URLs in SSG for this page, but that doesn't seem to help. Any tips are appreciated. Thanks!

Edit: Some digging revealed that this was an artifact of going from Statamic 5.3 to 5.6.2.

**Comment by joshuablum** [comment]
Closing this in favor of https://github.com/statamic/ssg/issues/174.
